### DISCUSSION TITLE
Where are you getting follow lists from?

### DISCUSSION THREAD

**Discussion by brianp** [obalunenko/instadiff-cli#283] [Q&A] [answered]
Hey!

Cool project. I'm wondering about the `follow-users` command. It accepts a comma separated list of user names. I'm assuming behind the scenes somewhere you're using a tool to generate this list? I'm curious about that 😄

**Comment by obalunenko** [comment, marked as answer]
Hye @brianp !
Thanks for your feedback! Unfortunately, there is no tool to generate such a list - I usually do it on my own by sneaking accounts related to my topics.
Maybe in the future I'll implement smth for that purpose 😄
